lib/mayaUsd/ufe/Utils.h

```diff
@@ -85,7 +85,7 @@ template <class T>
 void sendNotification(const Ufe::SceneItem::Ptr& item, const Ufe::Path& previousPath)
 {
     T notification(item, previousPath);
-    Ufe::Scene::notifyObjectPathChange(notification);
+    Ufe::Scene::instance().notify(notification);
```

Comment:
This class can now be used for any notification that takes an item and a path.  Probably only path change notifications will ever have this requirement, but in theory it's now a more general utility :)